packages/calcite-components/src/components/block/block.scss

```diff
@@ -150,18 +171,18 @@ calcite-handle {
 }
 
 .toggle-icon {
-  @apply text-color-3
-  transition-color
+  @apply transition-color
   self-center
   justify-self-end
   duration-150
   ease-in-out;
 
   margin-inline-end: var(--calcite-spacing-md);
+  color: var(--calcite-block-header-icon-color, var(--calcite-color-text-3));
```

Comment:
content, description, and icons all share this color. Therefore we can simplify it all down to a single "block-text-color" token which the icons will inherit via their default `currentColor`